flags/4x3/ws.svg

```diff
@@ -1,7 +1,7 @@
 <svg xmlns="http://www.w3.org/2000/svg" id="flag-icon-css-ws" width="640" height="480">
   <g fill-rule="evenodd" stroke-width="1pt">
-    <path fill="red" d="M0 0h640v480H0z"/>
-    <path fill="#00006b" d="M0 0h314.4v157.2H0z"/>
-    <path fill="#fff" d="M162.8 144.4l-12.5-8.4-12 8.5 3.9-15-11.5-9.2 14.8-.9 4.9-14.2 5.3 14.4 14.5.5-11.5 9.8zm-2.2-99.8l-10-6.4-9.7 6.3 2.7-12-8.8-7.7 11.7-1 4.3-11.2 4.5 11.3 11.5 1-8.9 8zm-44 35.9l-9.7-6.7-10 6.7 3.1-11.7-9.1-7.7 11.6-.6 4.2-11.5 4.1 11.4 11.8.7-9 7.5zm88.3-8l-9.3-6-9 6 2.6-11-8.4-7.1 11-.9 3.8-10.4 4.2 10.5 10.7.6-8.3 7.4zm-26 26.2l-6.2-3.9-6.2 4 1.6-7.4-5.5-5 7.2-.6 2.8-7 2.9 7 7.2.6-5.5 5z"/>
+    <path fill="#ce1126" d="m0 0h640v480h-640z"/>
+    <path fill="#002b7f" d="m0 0h392.9v260.14h-392.9z"/>
+    <path fill="#fff" d="m216.53 239.33-20.74-13.937-19.911 14.103 6.471-24.888-19.081-15.265 24.556-1.4933 8.1302-23.561 8.7939 23.893 24.059 0.82962-19.081 16.26zm-3.6503-165.59-16.592-10.619-16.094 10.453 4.4799-19.911-14.601-12.776 19.413-1.6592 7.1346-18.583 7.4665 18.749 19.081 1.6592-14.767 13.274zm-73.006 59.566-16.094-11.117-16.592 11.117 5.1436-19.413-15.099-12.776 19.247-0.99554 6.9687-19.081 6.8028 18.915 19.579 1.1615-14.933 12.444zm146.51-13.274-15.431-9.9553-14.933 9.9553 4.314-18.251-13.937-11.78 18.251-1.4933 6.305-17.256 6.9687 17.422 17.754 0.99553-13.772 12.278zm-43.14 43.472-10.287-6.471-10.287 6.6369 2.6548-12.278-9.1257-8.2961 11.946-0.99554 4.6458-11.615 4.8117 11.615 11.946 0.99554-9.1257 8.2961z"/>
```

Comment:
The blue part is taking a quarter of the area (half of the width and half of the height) in the original one.. can we keep this ratio?

Reply:
I've update the blue part to be a quarter of the area, I've been trying to match the flags as close to the original and not look too awkward. Are there any general guidlines for adjusting flags into 4:3 and 1:1 ratios?